configure.ac

```diff
@@ -61,6 +61,8 @@ if test "$fhs" = "yes"; then
     execcgidir=$libdir/sympa/cgi
     expldir=$localstatedir/lib/sympa/list_data
     staticdir=$localstatedir/lib/sympa/static_content
+    cssdir=$localestatedir/lib/sympa/static_content/css
+    picturesdir=$localestatedir/lib/sympa/static_content/pictures
```

Comment:
s/localestatedir/localstatedir/

Reply:
It seems I can't get these 2 lines right... :-( I fixed and force-pushed.